README.md

```diff
@@ -144,54 +144,36 @@ arguments can be given after `--`.
 
 ### cork download-image
 
-Download a CoreOS image into `$PWD/.cache/images`.
+Download a Container Linux image into `$PWD/.cache/images`.
 
 `cork download-image --platform=qemu`
 
-## building CoreOS on CoreOS
-
-Here is an example script that will download and build a CoreOS image using
-cork and the SDK. It is assumed that it is run on an existing CoreOS instance.
-
-The resulting QEMU images will be in
-`$HOME/coreos/src/build/images/amd64-usr/latest/`.
-
-```sh
-# setup env vars
-export PATH=$HOME/bin:$PATH
-export V=master
-export S=/mnt/host/source/src/scripts
-export B=amd64-usr
-
-# build mantle commands, including cork
-cd $HOME
-git clone https://github.com/coreos/mantle.git
-cd mantle
-docker run --rm -v "$PWD":/usr/src/myapp -w /usr/src/myapp golang ./build
-mkdir -p $HOME/bin/
-cp bin/* $HOME/bin/
-
-mkdir $HOME/coreos
-cd $HOME/coreos
-
-# download CoreOS SDK, build packages and image
-cork create --replace --sdk-version=${V} --verbose
-cork enter -- ${S}/build_packages --board=${B}
-cork enter -- ${S}/build_image --board=${B} dev prod
-
-# optionally you can run the kola tests on the built image
-cork enter -- sudo kola run -v
-
-# build dev and prod QEMU images
-cork enter -- ${S}/image_to_vm.sh --board=${B}
-cork enter -- ${S}/image_to_vm.sh --board=${B} --prod_image
-```
-
-## TODO
-
- - Migrate to the standard `log` package to make code easier to embed in
-   other projects which may not use `capnslog`.
- - Adopt `context` as the primary way of managing the life time of
-   asynchronous jobs. It is more standard and flexible than our local
-   `destructor` package.
- - Continue to expand the amount of data recorded in `_kola_temp`.
+### Building Container Linux with cork
+
+See [Modifying CoreOS](https://coreos.com/os/docs/latest/sdk-modifying-coreos.html) for
+an example of using cork to build a Container Linux image.
+
+## plume
+Container Linux release utility. Releases are done in two stages, each with their own command: pre-release
+and release. Both of these commands are idempotent.
+
+### plume pre-release
+The pre-release command does as much of the release process as possible without making anything public.
+This includes uploading images to cloud providers (except those like gce which don't allow us to upload
+images without making them public) and running kola tests against the uploaded images.
+
+## plume release
+Publish a new Container Linux release. This makes the images uploaded by pre-release public and uploads
+images that pre-release could not. 
+
+### plume index
+Generate and upload index.html objects to turn a Google Cloud Storage
+bucket into a publicly browsable file tree. Useful if you want something
+like Apache's directory index for your software download repository.
+
+## gangue
+Gangue is a tool for downloading and verifying files from Google Storage. It is primarily
+used by the SDK. 
+
+### gangue get
+Get a file from Google Storage and verify it using GPG.
```

Comment:
Maybe it's implicit, but it makes authenticated requests to Google Storage.  This is what allows our embargoed builds to work, so it might be worth mentioning.